RecoLocalCalo/HcalRecProducers/python/HcalHitSelection_cfi.py

```diff
@@ -11,5 +12,12 @@
                                          )
                                     )
 
+slimmedHcalRecHits = reducedHcalRecHits.clone(
+          hbheTag = cms.InputTag("reducedHcalRecHits","hbhereco"),
+          hfTag   = cms.InputTag("reducedHcalRecHits","hfreco"),
+          hoTag   = cms.InputTag(""),
+          interestingDetIds = cms.VInputTag()
```

Comment:
If you're overriding existing PSet items, it would be good to use the type-free syntax: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Modifying_Parameters

```python
hbheTag = "reducedHcalRecHits:hbhereco",
...
```